src/CountryFlag/index.js.flow

```diff
@@ -1,526 +1,18 @@
 // @flow
```

Comment:
What is the reason for getting rid off whole `codes` ?

Reply:
Where would you use that type?

Reply:
if you provide to `CountryFlag` wrong code flow error appears. We want to be strict what component could receive.

Reply:
@tomashapl but this PR is about relax on this strictness. It should be the `CountryFlag` responsibility to handle various input. It's the "be generous on input strict on output" principle.